### DISCUSSION TITLE
Scripts fails during installation, why?

### DISCUSSION THREAD

**Discussion by tomasenskede** [ej52/proxmox-scripts#110] [Q&A]
[success] Installing dependencies
[success] Installing python
[success] Installing openresty
[success] Installing nodejs
[success] Checking for latest NPM release
[success] Downloading NPM v2.10.1
[success] Setting up enviroment
[success] Generating dummy SSL certificate
[info] Building frontend
yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
rev: stdin: Invalid or incomplete multibyte or wide character

**Comment by superjma** [comment]
Maybe you are missing out of disk?
I had to give my image 4 GB instead of 2GB.
